themes/finna2/templates/Recommend/Ontology.phtml

```diff
@@ -0,0 +1,35 @@
+<?php if (!empty($ontologyResults = $this->recommend->getOntologyResults())): ?>
```

Comment:
Jos tuo palauttaa aina arrayn, niin riittää ` <?php if ($ontologyResults = $this->recommend->getOntologyResults()): ?>`

Reply:
Voi palauttaa myös `false` mutta ehdotettu muutos toimii senkin kanssa. 👍

Reply:
Muutan lisäksi samalla tavalla tehdyn `Feedback.html`:n.